src/modules/Categories.js

```diff
@@ -0,0 +1,227 @@
+
+    await result.then((response) => response.json()).then((json) => {
+      console.log(json.meals[0]);
+
+      const popupImg = this.popupComments.querySelector('#popupImage');
+      popupImg.src = json.meals[0].strMealThumb;
+
+      const popupTitle = this.popupComments.querySelector('#popupTitle');
+      popupTitle.innerHTML = json.meals[0].strMeal;
+
+      const gridContainer = this.popupComments.querySelector('.grid-container');
+      gridContainer.innerHTML = '';
+      let newData = document.createElement('div');
+      newData.innerHTML = `Area:${json.meals[0].strArea}`;
+      gridContainer.appendChild(newData);
+
+      newData = document.createElement('div');
+      newData.innerHTML = `Category:${json.meals[0].strCategory}`;
+      gridContainer.appendChild(newData);
+
+      const ingredientContainer = document.createElement('ul');
+      ingredientContainer.className = 'ingredient-container';
+      gridContainer.appendChild(ingredientContainer);
+      ingredientContainer.innerHTML = '';
+
+      newData = document.createElement('li');
+      newData.innerHTML = 'Ingredients:';
+      ingredientContainer.appendChild(newData);
+
+      const strIngredient = [];
+      for (let i = 1; i <= 20; i += 1) {
+        strIngredient[i] = json.meals[0][`strIngredient${i.toString()}`];
+        if (strIngredient[i] === '') { break; }
+        newData = document.createElement('li');
+        newData.innerHTML = strIngredient[i];
+        ingredientContainer.appendChild(newData);
+      }
+
+      const measureContainer = document.createElement('ul');
+      measureContainer.className = 'ingredient-container';
+      gridContainer.appendChild(measureContainer);
+      measureContainer.innerHTML = '';
+
+      newData = document.createElement('li');
+      newData.innerHTML = 'Maesures:';
+      measureContainer.appendChild(newData);
+
+      const strMeasure = [];
+      for (let i = 1; i <= 20; i += 1) {
+        strMeasure[i] = json.meals[0][`strMeasure${i.toString()}`];
+        if (strMeasure[i] === '') { break; }
+        newData = document.createElement('li');
+        newData.innerHTML = strMeasure[i];
+        measureContainer.appendChild(newData);
+      }
+
+      const instructions = document.querySelector('.instructions');
+      instructions.innerHTML = json.meals[0].strInstructions;
+
+      const comments = this.getComments(mealId);
+      this.printComments(comments);
+    });
+  };
+
+  displayPopupComments(mealId) {
+    this.popupComments.showModal();
+    this.getItemApiMain(mealId);
+
+    const addButton = document.querySelector('#add-button');
+    const form = document.querySelector('.input-section');
+
+    addButton.addEventListener('click', async () => {
+      const user = document.querySelector('.user').value;
+      const comment = document.querySelector('.comment').value;
+      await this.addComment(mealId, user, comment);
+      form.reset();
+      this.getItemApiMain(mealId);
```

Comment:
There's no need to re-render the whole Popup HTML every time you add a new comment, you can simply use the .appendChild method.